<ui version="4.0" >
 <class>DockNGListClass</class>
 <widget class="QDockWidget" name="DockNGListClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>259</width>
    <height>465</height>
   </rect>
  </property>
  <property name="features" >
   <set>QDockWidget::AllDockWidgetFeatures</set>
  </property>
  <property name="allowedAreas" >
   <set>Qt::AllDockWidgetAreas</set>
  </property>
  <widget class="QWidget" name="widget" >
   <layout class="QGridLayout" >
    <item row="0" column="0" >
     <widget class="QSplitter" name="splitter" >
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QTableWidget" name="dwTableList" />
      <widget class="QWidget" name="layoutWidget" >
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <widget class="QLabel" name="label" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>NG Count</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QLineEdit" name="leNGCount" >
          <property name="readOnly" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
